ChangeCalculation/ChangeCalculator.cs

```diff
@@ -1 +1,33 @@
-﻿
\ No newline at end of file
+﻿namespace ChangeCalculation
+{
+    public static class ChangeCalculator
+    {
+        private static readonly List<int> _validCoins = new List<int> { 100, 50, 10, 5, 2, 1 };
+
+        public static object CalculateChange(int change, List<int> availableCoins)
+        {
+            var invalidCoins = availableCoins.Where(x => !_validCoins.Contains(x)).ToList();
```

Comment:
Add multi-line here and in the similar places later like that:
availableCoins
    .Where(x => !_validCoins.Contains(x))
    .ToList();